src/main/java/shared/sharedObjects.java

```diff
@@ -9,7 +10,7 @@
   // public static
   static {
     try {
-      dataInstance = new Database("testdb-pose-3d-viewer-step");
+      dataInstance = new GCSDatabase("testdb-pose-3d-viewer-step");
```

Comment:
This class looks out of date. See my comment on David's PR for why we shouldn't have a shared database instance: https://github.com/googleinterns/step199-2020/pull/16#discussion_r468225590.

Reply:
ok thanks. i read thru it and will need to read again bc still confused about thread safety. but if i change it the way david did in  his branch, then i would have to declare a database in each servlet and the servlets would know directly what type of database is being implemented within it bc i would initialize the database as new GCSDatabase(); . I thought the point of having a shared objects file was to keep the code modular and to remove the detail of which database is being used (and also to keep it simple if database implementation needs to update) . will ping to learn more